### DISCUSSION TITLE
Tanstack table with filter and Sorting using Vannila JS/TS

### DISCUSSION THREAD

**Discussion by ibramahishab** [TanStack/table#5054] [Ideas]
I have tried implementing TanStack table [table-core] without any framework by using Vanilla JS. facing lot of challenges as there is no guidance or example specifically.

Is there anyone who implemented the tanStack table with sorting / filter functionality using Venilla JS.

Is implementation of table with sorting / filter functionality using Venilla JS - Possible or not. [ Any examples available]

**Comment by marceloverdijk** [comment]
You can find more info here: https://github.com/TanStack/table/discussions/4928
